src/apps/lwaftr/conf_parser.lua

```diff
@@ -231,6 +231,23 @@ function Parser:parse_string()
    return str
 end
 
+function Parser:parse_string_or_file()
```

Comment:
This should have been https://github.com/Igalia/snabb/pull/317/files#diff-ec8b020a9ff7c6f623ee90939c8a1ce1R234.